test-requirements.txt

```diff
@@ -1,3 +1,4 @@
+click
```

Comment:
This is what I was doing before I learned how to tell tox to install the package with various extras, I'll remove it :)